_includes/resource_type.html

```diff
@@ -2,15 +2,15 @@
     item.path contains 'shared' or item.path contains '{{ page.slug }}'
 {% endcapture %}
 
-{% assign resources = resources_by_type | where:"name", resource_type.slug | map:"items" | first %}
-
 <section>
     <header>
         <h3 id="{{ resource_type.slug }}">{{ resource_type.title }}</h3>
         <div class="buttons"><a href="#alternative-resources">Back to the top</a></div>
     </header>
 
-    {% for resource in resources %}
-        {% include resource.html %}
+    {% for resource in city_resources %}
+        {% if resource.categories contains resource_type.slug or resource.category == resource_type.slug %}
+            {% include resource.html %}
+        {% endif %}
```

Comment:
Does this change have an impact on page-load performance? Or rather, what step does this slow down, since I'm not too familiar with PWAs?

Reply:
It actually slows down site generation. The Liquid code is only run when the site is generated, not once it's loaded.

The performance concern is that if the site were to get much larger and have a lot more resources (probably like a LOT more) then the site generation (especially during development) could get slower.

But because it's a static site it'll never have a performance impact on the page load.